observability/power_monitoring/power-monitoring-release-notes.adoc

```diff
@@ -15,6 +15,38 @@ These release notes track the development of {PM-title} in the {product-title}.
 
 For an overview of the {PM-operator}, see xref:../power_monitoring/power-monitoring-overview.adoc#power-monitoring-about-power-monitoring_power-monitoring-overview[About {PM-shortname}].
 
+[id="power-monitoring-release-notes-0-2"]
+== {PM-shortname-c} 0.2 (Technology Preview)
+
+The following advisory is available for {PM-shortname} 0.2:
+
+* link:https://access.redhat.com/errata/RHEA-2024:2923[RHEA-2024:2923]
+
+[id="power-monitoring-release-notes-0-2-features"]
+=== Features
+
+* With this update, you can enable the Redfish API in {PM-kepler}. {PM-kepler} uses Redfish to get the power consumed by nodes.
```

Comment:
There should be a bit more information about the feature, or a link to more information.  For example, give at least one sentence on what Redfish is.  How do you enable the Redfish API?  Redfish isn't always available so which platforms will they be available on?

Reply:
Availability of redfish should already be known to the cluster admin.